lib/blob_store/Cargo.toml

```diff
@@ -0,0 +1,58 @@
+[package]
+name = "blob_store"
+version = "0.1.0"
+authors = ["Qdrant Team <info@qdrant.tech>"]
+license = "Apache-2.0"
+edition = "2021"
+publish = false
+
+[lints]
+workspace = true
+
+[features]
+bench_rocksdb = ["dep:rocksdb"]
+
+[dependencies]
```

Comment:
cargo machete says there are unused deps

```
        atomicwrites
        bincode
        log
        nix
        thiserror
```

Reply:
Thanks for running it, I forgot to remove these when removing the copied utilities